src/components/organisms/DisplayWalletBalance.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import { getWalletBalance } from "../../lib/brokerHandler";
+import TextDisplay from "../molecules/TextDisplay";
+
+const DisplayWalletBalance = () => {
+  const [walletBalance, setWalletBalance] = useState<string>("");
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const result = await getWalletBalance();
+      result && setWalletBalance(result);
```

Comment:
Does this mean that "getWalletBalance" could return "falsy" values?
What if the balance is "0"? Should we not display that as well?

If "getWalletBalance" fails for some reason, I suggest we let it throw an error/exception so that we can catch it here instead, and prompt a message.